### ISSUE TITLE
Question about multiple stations on one pi?

### ISSUE THREAD

**Issue by ryanginnelly252** [miegl/PiFmAdv#89]
With the --gpio command is it possible to have 2 stations with 1 Pi?

**Comment by miegl** [comment]
No, at least not yet. You can use the `--gpio` option multiple times to output the same signal on different pins.

**Comment by robinmeier** [comment]
just to say that this would be a cool feature indeed! would it be technically feasible?
